pkgs/by-name/ge/gem5/m5ops.nix

```diff
@@ -0,0 +1,53 @@
+{
+  fetchFromGitHub,
+  glibc,
+  isa ? "x86",
```

Comment:
I assume this is the simulation target? Are other ISAs available? please list them in a comment for both packages if yes